_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -470,6 +473,10 @@ public function get_all_options() {
 		return rest_ensure_response( $response );
 	}
 
+	private function decode_special_characters( $value ) {
```

Comment:
You could make this actually `static` 😁

Reply:
😶

Reply:
We're calling it only within this object, so should be good as-is, right?

Reply:
To provide some rationale for my thinking: I just like to "shrink-wrap" my API surface as tightly as possible around actual requirements. If this is a generic function by definition that makes no reference to `$this`, I think it's good practice to reflect that in the function's interface :slightly_smiling_face: